spark-publish/index.js

```diff
@@ -53,7 +53,7 @@ export const publish = async ({
     'measurements.json',
     { type: 'application/json' }
   )
-  const cid = await web3Storage.put([file])
+  const cid = await web3Storage.uploadFile(file)
```

Comment:
I noticed a change in behavior here, see https://filecoinproject.slack.com/archives/C027XP5BTGB/p1700818334024649

Reply:
This new one is actually simpler, since we don't need to support the extra complexity of unixfs